core/templates/dev/head/editor/ExplorationFeedback.js

```diff
@@ -161,23 +236,48 @@ oppia.controller('ExplorationFeedback', [
     {id: 'not_actionable', text: 'Not Actionable'}
   ];
 
-  $scope.getHumanReadableStatus = function(status) {
-    for (var i = 0; i < $scope.STATUS_CHOICES.length; i++) {
-      if ($scope.STATUS_CHOICES[i].id === status) {
-        return $scope.STATUS_CHOICES[i].text;
+  return {
+    STATUS_CHOICES: angular.copy(_STATUS_CHOICES),
+    getLabelClass: function(status) {
```

Comment:
Hm, I don't think I want to do that; it feels a little ugly to have so much logic in the HTML. It also makes it harder to test.